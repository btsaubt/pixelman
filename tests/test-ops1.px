def int main()
{
  print_int(1 + 2);
  print_int(1 - 2);
  print_int(1 * 2);
  print_int(100 / 2);
  print_int(99);
  printb(1 == 2);
  printb(1 == 1);
  print_int(99);
  printb(1 != 2);
  printb(1 != 1);
  print_int(99);
  printb(1 < 2);
  printb(2 < 1);
  print_int(99);
  printb(1 <= 2);
  printb(1 <= 1);
  printb(2 <= 1);
  print_int(99);
  printb(1 > 2);
  printb(2 > 1);
  print_int(99);
  printb(1 >= 2);
  printb(1 >= 1);
  printb(2 >= 1); 
  return 0;
}
