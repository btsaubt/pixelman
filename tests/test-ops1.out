3-125099FalseTrue99TrueFalse99TrueFalse99TrueTrueFalse99FalseTrue99FalseTrueTrue